src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.state.Figure;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int baseA;
+    private int baseB;
+    private int height;
+
+    public IsoscelesTrapezoid(String name) {
+        super(name);
+        baseA = (int)((Math.random() * 10) + 1);
+        baseB = (int)((Math.random() * 10) + 1);
+        height = (int)((Math.random() * 10) + 1);
```

Comment:
Do not initialize variables with random values in the constructor. Pass them from the outside

Reply:
done